### ISSUE TITLE
Burnt manager

### ISSUE THREAD

**Issue by waltiz** [davidsansome/tsurukame#350]
It would be great to have a burnt manager within the app. It could propose to revive items after X months they were burnt or if some conditions are met (for example getting confused one kanji with a similar one that was burnt or a word that had a burnt kanji). If this is too complicated it would be helpful at least to browse burnt items manually and change their status and maybe being randomly quizzes on them.

**Comment by bmironer** [comment]
seconded. I'd love to be able to start a review of the burnt items 'just to check'.
And if WaniKani supports it maybe downgrade the wrong answers to enlightened?

I've been a bit lazy lately and I feel reviewing these older items might be worth it.

**Comment by turtlemaster19** [comment]
@waltiz @bmironer Possible dupe of #257

It's not possible in WaniKani, although it has been requested often, to resurrect a burned item to any stage other than Apprentice I.

**Comment by waltiz** [comment]
Sadly I know it’s not possible to resurrect items to enlightened (I also requested it to Sabina I), but I’d rather have them in apprentice rather then forget them altogether. I would only do it when I realise that I genuinely struggle with some item

**Comment by turtlemaster19** [comment]
@waltiz So this is an additional request after #257 is complete, because you want an optional burn review that can be completed (after your real ones of course) for burns older than, say, 6 (possibly customizable between 3-10) months. You would be able to enter the date that you believe you did your last burn review (for changing device)

If you get an item **wrong** then you would know to go to the item page at the end-of-review items incorrect list and resurrect it.

If right, it would stay burned (unless you wanted to resurrect it anyways?!?) and not appear in another burn review for this interval.

Does that sound about right?

**Comment by waltiz** [comment]
Yes this sounds about right. Ideally as you say burnt reviews would be after real reviews. The summary at the end should make clear which failed reviews were ‘normal’ and the failed burnt should be grouped at the end, with a button to resurrect individual items or all at once.

**Comment by UInt2048** [comment]
If we ever do this, we could store data in the meaning note for 火 with https://github.com/qntm/base32768 or something...

**Comment by crun** [comment]
i recently posted to the wanikani community about a burned items version of tsurukame. my wish was for a separate app, perhaps one that allows you to chose which class of burned items to review: radicals, kanji, vocabulary. i'd especially like the option of ignoring radicals because wanikani's bespoke radicals are bullshit. there are only a small number of radicals in japanese, not the hundreds wanikani wants you to memorise.

however, if you're considering adding this as an option to the tsurukame review workflow, i totally support the notion and would be pleased to help validate it.

**Comment by waltiz** [comment]
> @crun You might want to use Anki at this point, since WK doesn’t really support this functionality.

This seems a non sequitur. WK doesn’t support  many of Tsurukame’s features, that didn’t prevent these features from being implemented in the app.

**Comment by crun** [comment]
> @crun You might want to use Anki at this point, since WK doesn’t really support this functionality.

i'm sorta stuck with WK at this point. the ability to filter out their bs radicals is something that would be nice to have but not essential. being able add burnt items to reviews is something i'd really like to have. i'll take the good over the perfect.

thanks so much for considering this feature.

**Comment by crun** [comment]
further thoughts on this request: the way wanikani implements their burned items extra study is exactly the feature i desire, just with the better u/i of tsurukame. mingling burned items in with the regular review doesn't seem the right approach in my opinion.

i also really like that tsurukame allows me to cheat, which is how i deal with wanikani's bogus radicals: "my answer was correct!" frankly that's more than good enough.
